pkg/lib/authn/user/store.go

```diff
@@ -285,6 +287,28 @@ func (s *Store) QueryPage(listOption ListOptions, pageArgs graphqlutil.PageArgs)
 	return users, offset, nil
 }
 
+func (s *Store) QueryForExport(offset uint64) ([]*User, error) {
```

Comment:
Take limit as argument. In general, pkg/lib/authn/user is a general package that will be depended by many packages. So it should not import userexport. userexport is a package for the user export feature, so it is not a general package.

Reply:
added limit argument for the query